### PR TITLE
add access logs for passthrough listener

### PR DESCRIPTION
**Pull request by dwradcliffe**
We would like to have logs for requests going through the passthrough listener/cluster. This will help us transition onto the mesh and identify traffic that is not yet fully onboarded.

Also ref: https://discuss.istio.io/t/telemetry-metrics-logging-on-passthroughcluster/2005

### CODE REVIEW

File: pilot/pkg/networking/core/v1alpha3/listener.go [hunk 1]
```diff
@@ -213,6 +213,7 @@ func (configgen *ConfigGeneratorImpl) buildSidecarListeners(env *model.Environme
 				StatPrefix:       util.PassthroughCluster,
 				ClusterSpecifier: &tcp_proxy.TcpProxy_Cluster{Cluster: util.PassthroughCluster},
 			}
+			setAccessLog(env, node, tcpProxy)
```
Comment (howardjohn) [inline pilot/pkg/networking/core/v1alpha3/listener.go]: Should we add it for blackhole cluster as well?

Comment (rshriram) [reply pilot/pkg/networking/core/v1alpha3/listener.go]: It’s not about the cluster. It’s about the listeners. And currently this is the only wildcard listener in the system. All others have access logs iirc. If that’s not the case, then yeah.

Comment (dwradcliffe) [reply pilot/pkg/networking/core/v1alpha3/listener.go]: There's also a fallback route, that might be used on ports that have other services, but that should be covered by the existing log config on the listener.

**Comment by douglas-reid** [comment]
Do we need some sort of integration / e2e test that validates we are seeing a desired behavior here?  I don't see any tests in this PR.

**Pull request merged**